src/coordinatesystems.jl

```diff
@@ -73,6 +73,26 @@ Base.convert(::Type{Polar}, v::AbstractVector) = PolarFromCartesian()(v)
 #############################
 """
 Spherical(r, θ, ϕ) - 3D spherical coordinates
+
+This uses the so-called mathematics convention as opposed to the physics convention.
+Given the Cartesian coordinates `x,y,z` of a vector `v`, the polar coordinates are:
+
+* `r` is the radius. It is given by `norm(v, 2)`.
+* `θ` is the azimuthal angle. It is the angle from the x-axis to the projection of `v` onto the xy-plane.
+* `ϕ` is the polar angle. It is the angle from the z-axis to `v`.
```

Comment:
In particular, IIRC we had a somewhat dodgy/weird definition and `ϕ` was the latitude rather than the polar angle.

Reply:
Dang you are right!